lint-rules/src/main/java/com/duckduckgo/lint/ui/NoAlertDialogDetector.kt

```diff
@@ -73,7 +73,7 @@ class NoAlertDialogDetector : Detector(), SourceCodeScanner {
                 androidSpecific = true,
                 implementation = Implementation(
                     NoAlertDialogDetector::class.java,
-                    Scope.RESOURCE_FILE_SCOPE
+                    Scope.JAVA_FILE_SCOPE
```

Comment:
I don't know what this change does, so just flagging to ensure it was deliberate.

Reply:
In Lint terms, the scope defines when is this rule being inspected. I had this wrongly set to resource files, so Lint wasn't really working. `JAVA_FILE_SCOPE` applies to all source files, Java and Kotlin, which is what we want.